src/layer.rs

```diff
@@ -72,6 +83,10 @@ impl Layer {
         let path = path.as_ref();
         fs::create_dir(&path)?;
         plist::to_file_xml(path.join(CONTENTS_FILE), &self.contents)?;
+        // Avoid writing empty layerinfo.plist file.
+        if self.info.color.is_some() || self.info.lib.as_ref().map_or(false, |v| !v.is_empty()) {
```

Comment:
totally not a blocker but I would normally add a convenience method to `FontInfo` for this, and then we can just do, `if self.info.has_contents() {` or something similar.

Reply:
Made a `is_empty`!